lms/templates/admin/macros.jinja

```diff
@@ -131,4 +140,28 @@
             {{ disabled_text_field("Client ID", lti_registration.client_id) }}
         </div>
     </div>
+    {% else %}
+    <div class="is-size-5 has-text-centered">No LTI registration</div>
```

Comment:
Adding the message instead of having blank fields on the existing LTIRegistraion marco